web/modules/custom/joinup_core/joinup_core.drush.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Drush commands related to joinup.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function joinup_core_drush_command() {
+  $items = [];
+
+  $items['joinup-status-check'] = [
```

Comment:
This drush command is a duplicate of `drush core-requirements --severity=2`, which also has a better output.